linode/manager
# fix: [M3-8657] - Set full height to DX Tools Modal and add Linode API link

## Description 📝
Add Linode API link to the DX Tools and Set full height to the DX tools Modal.


## Target release date 🗓️
9/30

## Preview 📷
**Include a screenshot or screen recording of the change**
![image](https://github.com/user-attachments/assets/f36a818f-cab6-4fcb-9f3d-6aa272bbb1f1)


## How to test 🧪

### Verification steps
(How to verify changes)
- Navigate to DX Tools Modal
- Verify Linode API Link into into text and Modal height.
- Verify no regressions around DX tools.

## As an Author I have considered 🤔

*Check all that apply*
